lib/src/front_end/comment_writer.dart

```diff
@@ -106,6 +106,7 @@ mixin CommentWriter {
     // just override the script tag's line.
     if (token.previous!.type == TokenType.SCRIPT_TAG) previousLine = tokenLine;
 
+    // ignore: prefer_const_constructors
```

Comment:
Does the linter really yell at you to use `const` here?! It's creating a mutable object. That's a bananas lint. I definitely don't want to enable it for dart_style.

Reply:
Yep!

If we don't want it enabled, there's a lot of churn to delete.